README.md

```diff
@@ -14,6 +14,46 @@ Newman Dashboard aims to provide a real-time WebUI dashboard which can let you c
 
 This project was built during [Google Summer of Code 2021](https://summerofcode.withgoogle.com/projects/#5547391014404096) ☀️ with [Postman](https://github.com/postmanlabs) 🚀
 
+
+## Usage
+
+This package is both a reporter and a standalone package. The standalone package takes care of launching the dashboard, while the reporter connects to the newman run.
+
+A. For installation:
+
+```
+npm install -g newman-dashboard
```

Comment:
We should remove this because we've not published the package. See #51

Reply:
Actually source code for reporter is already there.
All we have to do is execute just one more command before this.

`npm pack reporter`
This will generate a .tgz file, in my case it was `newman-reporter-dashboard-0.0.1.tgz`
then we can use
`npm i -g newman-reporter-dashboard-0.0.1.tgz` to install the package.

Reply:
I am working on this give me some time